src/main/scala/xiangshan/backend/fu/NewCSR/NewCSR.scala

```diff
@@ -1048,13 +1048,6 @@ class NewCSR(implicit val p: Parameters) extends Module
     hcEvents(i) := io.perf.perfEventsHc(i)
   }
 
-  val allHcPerfEvents = hcEvents.map(x => (s"Hc", x.value))
-  if (printEventCoding) {
-    for (((name, inc), i) <- allHcPerfEvents.zipWithIndex) {
-      println("HuanCun perfEvents Set", name, inc, i)
```

Comment:
Why remove this？

Reply:
This output isn't clear enough. It will say the name of some HPM is `Hc1` instead of its actual name.

Reply:
A new version of printing L2 HPM will be updated after CoupledL2's update.

Reply:
https://github.com/search?q=repo%3AOpenXiangShan%2FXiangShan%20printEventCoding&type=code
Moreover, other events are printed in [Frontend.scala](https://github.com/OpenXiangShan/XiangShan/blob/a1c30bb906ea7a5453128e560eb608d5cf4d2a24/src/main/scala/xiangshan/frontend/Frontend.scala#L366) , [MemBlock.scala](https://github.com/OpenXiangShan/XiangShan/blob/a1c30bb906ea7a5453128e560eb608d5cf4d2a24/src/main/scala/xiangshan/backend/MemBlock.scala#L1818) and [Backend.scala](https://github.com/OpenXiangShan/XiangShan/blob/a1c30bb906ea7a5453128e560eb608d5cf4d2a24/src/main/scala/xiangshan/backend/Backend.scala#L764). It's strange to print L2 Cache's HPM name and code here. L2Top.scala may be proper.